__tests__/cli-test.ts

```diff
@@ -37,11 +40,29 @@ describe('cli-test', () => {
     `);
   });
 
-  it('can invoke the preflight command', async () => {
+  it('can invoke the preflight command using the default markdown formatter', async () => {
     const result = await run(['preflight']);
 
     expect(result.exitCode).toEqual(0);
-    expect(JSON.parse(result.stdout)).toBeValidSarifLog();
+    expect(result.stdout).toMatch(
+      /Embroider preflight check written to .*\/embroider-preflight.md/
+    );
+  });
+
+  it('can invoke the preflight command using the pretty formatter', async () => {
+    const result = await run(['preflight', '--format', 'pretty']);
+    debugger;
```

Comment:
```suggestion
```

Surprised this didn't fail linting 🤔

Reply:
Gah. Indeed. I'll take a look at linting setup.